src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorSymbolReferenceFinder.cs

```diff
@@ -113,91 +114,100 @@ private static bool IsPotentialReference(PredefinedType predefinedType, ISyntaxF
                 continue;
 
             await AddReferencesInDocumentWorkerAsync(
-                methodSymbol, globalAlias, state, result, cancellationToken).ConfigureAwait(false);
+                methodSymbol, globalAlias, state, processResult, processResultData, cancellationToken).ConfigureAwait(false);
         }
 
         // Nest, our containing type might itself have local aliases to it in this particular file.
         // If so, see what the local aliases are and then search for constructor references to that.
         using var _2 = ArrayBuilder<FinderLocation>.GetInstance(out var typeReferences);
         await NamedTypeSymbolReferenceFinder.AddReferencesToTypeOrGlobalAliasToItAsync(
-            methodSymbol.ContainingType, state, typeReferences, cancellationToken).ConfigureAwait(false);
+            methodSymbol.ContainingType, state, StandardCallbacks<FinderLocation>.AddToArrayBuilder, typeReferences, cancellationToken).ConfigureAwait(false);
 
-        var aliasReferences = await FindLocalAliasReferencesAsync(
-            typeReferences, methodSymbol, state, cancellationToken).ConfigureAwait(false);
+        // REVIEW: This looks to go to quite a bit of work to populate aliasReferences, but never uses it?
+        using var _3 = ArrayBuilder<FinderLocation>.GetInstance(out var aliasReferences);
+        await FindLocalAliasReferencesAsync(
+            typeReferences, methodSymbol, state, StandardCallbacks<FinderLocation>.AddToArrayBuilder, aliasReferences, cancellationToken).ConfigureAwait(false);
```

Comment:
i think i'm missing where aliasreferences ends up getting used.

Reply:
oh.  looks like you that was yoru comment as well :D.  yes.  seems like we can jsut remove this.  if no tests break, we're good.